### ISSUE TITLE
About od model to generate coco_caption features. I cannot reproduce your feature results.

### ISSUE THREAD

**Issue by 309018451** [microsoft/Oscar#181]
I tried to reproduce the results for VinVL+SCST on NoCaps, but my result was also off by a visible margin.

Then I generate nocpas features by od model in pretrained models at [vinvl_vg_x152c4.pth](https://github.com/microsoft/Oscar/issues/131#issue-971135830](https://penzhanwu2.blob.core.windows.net/sgg/sgg_benchmark/vinvl_model_zoo/vinvl_vg_x152c4.pth), when I simplely check the total types of generated tags train.label.tsv, it has totally 1319 types. I can only get 12.1 SPICE scores after CC(Cross entropy)+CIDER OPTIM. The config file is [val_vinvl_x152c4.yaml](https://github.com/microsoft/Oscar/issues/131#issue-971135830](https://github.com/309018451/Oscar/blob/main/val_vinvl_x152c4.yaml)

For the coco_caption training set, train.label.tsv has totally 435 types. I can normally reproduce your result 12.8 SPICE.

May I ask, which model should I use to reproduce your od feature results, can you give me a link to this od model (To reproduce your feature results and similarly generate them) ?
If it's not available, Or I should try a od model on visual genome and open images by myself?

**Comment by xiaoweihu** [comment]
Hi, the od features are generated from the model you linked [vinvl_vg_x152c4.pth](https://penzhanwu2.blob.core.windows.net/sgg/sgg_benchmark/vinvl_model_zoo/vinvl_vg_x152c4.pth).
If you need the features on COCO or nocaps, you can download the pre-extracted features and labels at https://github.com/microsoft/Oscar/blob/master/VinVL_DOWNLOAD.md#datasets

**Comment by pzzhang** [comment]
@309018451  For the image tags, the image tags used in both NoCaps training and testing are generated from an OD model pretrained on OpenImages dataset, not from the model vinvl_vg_x152c4.pth. 

We cannot release the OD model pretrained on OpenImages dataset, but you should be able to train your own OpenImages OD model or use some public OpenImages OD model.

**Comment by 309018451** [comment]
@pzzhang @xiaoweihu
Thank you so much!
I'll try and make it work in my case.
Again, thank you a lot 💯
